package.json

```diff
@@ -16,5 +16,8 @@
     "gulp-sourcemaps": "^2.6.4",
     "postcss": "^6.0.22",
     "yargs": "^6.4.0"
+  },
+  "dependencies": {
```

Comment:
this is because this error when running grunt, https://github.com/nodejs/node/issues/25132